src/stories/radioButton.stories.tsx

```diff
@@ -13,6 +13,7 @@ export default {
   title: "Radio Button",
   component: RadioButton,
   args: {
+    name: "radioButton",
```

Comment:
you didn't change anything to the props - how did this not throw a type error? shouldn't the Prop types have enforced passing a name here?

Reply:
I tried with other components such as AlertMessage. I removed the `type` (which is a mandatory prop) from the stories and it does not complain 🤯

Reply:
Weird! Let's flag this mentally - if we don't figure it out on the go, we can follow up with a proper task